{"uid":"6c0e4e435ec446b0","name":"Oops...we found an error preparing this test file: cypress/e2e/first.cy.js","fullName":"Oops...we found an error preparing this test file: cypress/e2e/first.cy.js","time":{"start":1686596346860,"stop":1686596346860,"duration":0},"status":"broken","statusMessage":"Oops...we found an error preparing this test file:\n\n  > cypress\\e2e\\first.cy.js\n\n","statusTrace":"\n\nError: Looked for and couldn't find the file at the following paths:ilation Error\nmulti ./cypress/e2e/first.cy.js\nModule not found: SyntaxError: D:\\VSC\\DemoQAtest\\package.json (directory description file): SyntaxError: Unexpected token } in JSON at position 238\n \n    at handle (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\@cypress\\webpack-preprocessor\\dist\\index.js:212:23)\n    at finalCallback (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:257:39)\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:306:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:304:22\n    at Compiler.emitRecords (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:499:39)\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:298:10\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:485:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:482:27\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:2818:7\n    at done (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:3522:9)\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:464:33\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\НР\\AppData\\Local\\Cypress\\Cache\\12.14.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:61:14\n    at FSReqCallback.oncomplete (node:fs:188:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"cypress/e2e/first.cy.js"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"6c0e4e435ec446b0.json","parameterValues":[]}